aws/hhvm1/state-machine/preview.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+$(dirname $0)/generate.hack | jq -C . | less -R
```

Comment:
"$(dirname "$0")"

Reply:
Oh cool, I didn't know you can have nested quotes.